src/foam/u2/detail/SectionView.js

```diff
@@ -78,6 +78,17 @@ foam.CLASS({
           return this.getNestedPropValue(obj[props.shift()], props.join('.'))
         }
       }
+    },
+    {
+      class: 'Boolean',
+      name: 'loadLatch',
+      preSet: function(o, n) {
+        return this.sectionContentViewSet ? o || n : n;
+      }
```

Comment:
so default value - false and we load the view on false because if SectionView.js is used elsewhere other then in TabbedDetailVIew we should load at

value is false but then changed to true in TabbedDetailVIew if tab isn't selected and we need to ignore it